templates/default/rundeck-upstart.conf.erb

```diff
@@ -0,0 +1,29 @@
+# RunDeck
```

Comment:
It has a template logic in it. Check the line https://github.com/nilroy/rundeck/blob/2018dfc7219cd187109fca9f3e21dfd1ae4c4413/templates/default/rundeck-upstart.conf.erb#L26